src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { ResumeContext } from "./utils/ResumeContext";
+import "./App.css";
+import Router from "./routes/router";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+const App = () => {
+  const queryClient = new QueryClient();
+  const [initialState, setInitialState] = useState({
```

Comment:
can we add redux to maintain data ? as we need to use data fetched from API at multiple places.

https://redux.js.org/introduction/getting-started

Reply:
Discussed Offline